server/metrics/size.go

```diff
@@ -19,15 +19,19 @@ import (
 )
 
 var (
-	NamespaceSize  tally.Scope
-	DbSize         tally.Scope
-	CollectionSize tally.Scope
+	NamespaceSize   tally.Scope
+	DbSize          tally.Scope
+	CollectionSize  tally.Scope
+	SearchSize      tally.Scope
+	SearchIndexSize tally.Scope
 )
 
 func initializeSizeScopes() {
 	NamespaceSize = SizeMetrics.SubScope("namespace")
 	DbSize = SizeMetrics.SubScope("db")
 	CollectionSize = SizeMetrics.SubScope("collection")
+	SearchSize = SizeMetrics.SubScope("search")
+	SearchIndexSize = SizeMetrics.SubScope("search-index")
```

Comment:
I think the minus here is an invalid character in the metric name, and this is why you see sometimes successful and sometimes failing test. If this scope is used, tally will panic, if not, it will not. The name of the final metric is the scopes concatenated with an `_` character. Can you try and fix this and let is know how it goes?